src/next-js/logger.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Initialize Tidewave logging for Next.js applications.
+ * This function should be called from your `register` function in instrumentation.ts.
+ *
+ * @example
+ * ```typescript
+ * // instrumentation.ts
+ * import { registerTidewaveLogger } from 'tidewave/next-js/logger'
+ *
+ * export function register() {
+ *   registerTidewaveLogger()
+ *   // other instrumentation...
+ * }
+ * ```
+ */
+export async function registerTidewaveLogger(): Promise<void> {
+  const env = process.env['NODE_ENV'];
+  const runtime = process.env['NEXT_RUNTIME'];
+
+  if (env === 'development' && runtime === 'nodejs') {
```

Comment:
We rely on the user doing this. We can still do the conditional here, but we should raise if it is not what we expect.